data-prepper-expression/src/test/java/org/opensearch/dataprepper/expression/GenericExpressionEvaluator_MultiTypeIT.java

```diff
@@ -28,14 +27,12 @@
 import static org.hamcrest.CoreMatchers.equalTo;
 import static org.hamcrest.CoreMatchers.instanceOf;
 import static org.hamcrest.CoreMatchers.is;
-import static org.hamcrest.CoreMatchers.isA;
 import static org.hamcrest.CoreMatchers.not;
-import static org.hamcrest.CoreMatchers.sameInstance;
 import static org.hamcrest.MatcherAssert.assertThat;
 import static org.junit.jupiter.api.Assertions.assertThrows;
 import org.apache.commons.lang3.RandomStringUtils;
 
-class GenericExpressionEvaluator_StringIT {
+class GenericExpressionEvaluator_MultiTypeIT {
```

Comment:
This test was explicitly named `_StringIT`. This may be an indication that we should have a different IT for `_MultiTypeIT`. Would it duplicate a lot of code?

Thoughts?